### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by MustafaSamiShirzad**
added comments

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```
Comment (MustafaSamiShirzad) [inline index.html:5]: great work using semantics.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  justify-content: center;
+  flex-direction: column;
+  padding-bottom: 15px;
+  border-radius: 5px;
+  min-height: 100vh;
+}
+.main {
+  height: 20vh;
+  display: flex;
+  flex-direction: column;
```
Comment (MustafaSamiShirzad) [inline main.css:14]: box-model is used  perfectly

File: index.html (hunk 1 above)
Comment (stanleySimeon) [reply index.html:5]: Great work living comment as required but there are some changes you still need to perform.
Kindly, I suggest you use polite suggestions, like: `Great work using semantic tags`.

File: main.css (hunk 2 above)
Comment (stanleySimeon) [reply main.css:14]: As mentioned above great work living comment as required but there are some changes you still need to perform.
Kindly, I suggest you use polite suggestions, like: `box model is used perfectly! 😄`.

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @MustafaSamiShirzad, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- adding your readme file
- adding review in your `CSS` file
- adding a suggestion for your `PR` description

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  align-items: center;
+  justify-content: center;
+}
+
+.tribute_caption {
+  padding: 10px;
+}
+.tribute_life {
+  display: flex;
```
Comment (MustafaSamiShirzad) [inline main.css:37]: Great that you have used flex box.

**Review by MustafaSamiShirzad — COMMENTED** [review]
Look very close just add README file. AND GOOD WORK KEEP IT UP.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @MustafaSamiShirzad, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**